src/Web/Base/EntityModificationAction.php

```diff
@@ -27,10 +27,9 @@
     abstract protected function initModel();
 
     protected function loadModelAndSaveOrPrintView() {
-        $isSaved = $this->loadModelAndSave();
-        $model = $this->getModel();
-
-        return $isSaved && $this->redirectUrl !== false ? $this->redirectToNextPage() : $this->renderViewFile(compact('model'));
+        return $this->loadModelAndSave() ?
```

Comment:
After another look, I think maybe it would be better to have this structured in the following way:
```
if modelLoaded
   saveModel
else 
   printView
```
Then in `saveModel` have the condition we have right now in `loadModelAndSaveOrPrintView` except the need to load model.
`printView` can duplicate `handleFailedOperation` because it's going to be different use-cases and two methods can exist for extendibility.